### PR TITLE
134: Add Redis service to the platform.sh.

### PR DESCRIPTION
**Pull request by Szarad**
Add Redis service to dev environment.

### CODE REVIEW

**Comment by lsmith77** [comment]
@Szarad I think you are missing the change to `.platform/services.yml` to enable the redis service

**Comment by Szarad** [comment]
What do you mean @lsmith77 ? The change is in .platform/services.yml and Redis service works (on PR env). 
Sorry for not putting it as draft,  I forgot.

**Comment by lsmith77** [comment]
ah .. sorry .. misread. you are correct!

**Comment by Szarad** [comment]
To do:
1. Add necessary libraries for tests not to fail.
2.  Add new unit tests
3. Change company<->user.id relationship
4. Optional: changes in the logic
5. Exception handling

**Comment by Szarad** [comment]
I just noticed that I had a plugin turned on, that reformatted the whole file. If it is too difficult to review with all these stylistic changes, I can commit again. Sorry for that.

File: Pipfile [hunk 1]
```diff
@@ -15,6 +15,10 @@ aiofiles = "*"
 babel = "*"
 aiohttp = "*"
 sentry-sdk = "*"
+platformshconfig = "*"
+redis = "*"
+fakeredis = "*"
```
Comment (lsmith77) [inline Pipfile]: I guess these two dependencies should be moved to the dev-packages section

File: app/main.py [hunk 2]
```diff
@@ -46,6 +49,16 @@
 from pydantic import BaseSettings
 from app.categories import categories
 
+
+from redis import Redis
+#from platformshconfig import config
```
Comment (lsmith77) [inline app/main.py]: I guess this can be removed

File: app/main.py [hunk 3]
```diff
@@ -179,14 +197,25 @@ async def custom_http_exception_handler(request, e):
               "Kollektivgesellschaft", "Team", "Organization", "Gliederung"]
 gender_false_positive = genderdenom_false_positives["False_positives"].tolist()
 
-#corporate false positive
-#DB
-corporate_false_positive = ["stark", "starke", "starkes", "starker", "Führungskraft", "Führungskräfte", "Führungskräften"]
+# corporate false positive
+# DB
+corporate_false_positive = ["stark", "starke", "starkes",
```
Comment (lsmith77) [inline app/main.py]: this can be removed, ie. we don't need the hardcoded false positive list anymore

File: Pipfile (hunk 1 above)
Comment (Szarad) [reply Pipfile]: good point, yes fakeredis and pytest-asyncio  is need for testing, I'll move it

File: app/main.py (hunk 2 above)
Comment (Szarad) [reply app/main.py]: What exactly do you mean?
I can remove import json  and JSONResponse as it is not used. I can also remove commented imports.

File: app/main.py (hunk 3 above)
Comment (Szarad) [reply app/main.py]: I think I don't understand. 
I thought these are the only options:

    store_context: Optional[bool] = True
    primary_language: Optional[str] = "de-DE"
    preferred_languages: Optional[str] = "de,en"
    preferred_variants: Optional[str] = "de-DE,en-GB"
    german_gender_ending: Optional[str] = ":in"
   _gendereddenom_ending = {"/in": "/in", "/-in": "/-in", "_in": "_in", "*in": "\\*in", ":in": ":in", "In": r"In\b"}
   disabled_categories: Optional[List] = ""
   gendered_roles_format: Optional[GenderedRolesFormatType] = "inclusive_gender"

Where user can store false positive ?

File: app/main.py (hunk 2 above)
Comment (lsmith77) [reply app/main.py]: just that we don't need to keep commented out code

File: app/main.py (hunk 3 above)
Comment (lsmith77) [reply app/main.py]:
```
corporate_false_positive = ["stark", "starke", "starkes", "starker", "Führungskraft", "Führungskräfte", "Führungskräften"]
```

These are "false positives" we hardcoded for DB. this list needs to be customizable by corporation in the dashboard and then synced via your API into Redis

Comment (Szarad) [reply app/main.py]: ah ok it is just for Redis? I will then add another class based on Config with attribute corporate_false_positive, so when dashboard triggers POST on redis it know what to expect.

Comment (lsmith77) [reply app/main.py]: yes .. we will not allow this config via the browser extension, ie. it cannot be submitted via a /check request, it has to come from Redis (we use an empty `[]` as default)

Comment (lsmith77) [reply app/main.py]: moved to https://github.com/witty-works/nlp_api/issues/198

**Pull request merged**